_posts/2019-08-20-service-objects.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title:  "Service Objects: beyond fat models and skinny controllers"
+date: 2019-08-20 10:00:00
+categories: ["rails"]
+author: "cleiviane"
+---
+
+Service Objects is a controversial pattern for several different reasons: some developers like to use it, others like to use similar patterns and some think that they are just unnecessary due that fat models approach recommended to Rails apps.  
```

Comment:
```suggestion
Service Objects are a controversial idea for several different reasons: some developers like to use them, others like to use similar patterns, and some think that they are just unnecessary because they prefer fat models.  
```